mongodbatlas/resource_mongodbatlas_cluster_test.go

```diff
@@ -305,6 +305,7 @@ func TestAccResourceMongoDBAtlasCluster_basicAzure(t *testing.T) {
 }
 
 func TestAccResourceMongoDBAtlasCluster_basicGCP(t *testing.T) {
+	SkipTestExtCred(t)
```

Comment:
Interesting - this is another one I'm surprised to see peering in the pre check.  If it's a basic GCP cluster I don't see why we'd have peering involved - i.e. TestAccResourceMongoDBAtlasCluster_withGCPNetworkPeering should cover it.